src/ls/Lsp_util.mli

```diff
@@ -0,0 +1,2 @@
+val range_of_location : Semgrep_output_v1_t.location -> Lsp.Types.Range.t
+val string_of_severity : Rule.severity -> Lsp.Types.DiagnosticSeverity.t
```

Comment:
This does not return a strings maybe lsp_severity_of_rule_severity
(the final type may be a string, but this is an implementation detail we should hide. You use DiagnosticSeverity.Error
in the code, so this looks like a real type)